modules/apps/27-interchain-accounts/controller/ibc_module.go

```diff
@@ -180,5 +156,5 @@ func (im IBCModule) NegotiateAppVersion(
 	counterparty channeltypes.Counterparty,
 	proposedVersion string,
 ) (string, error) {
-	return im.keeper.NegotiateAppVersion(ctx, order, connectionID, portID, counterparty, proposedVersion)
+	return "", sdkerrors.Wrap(types.ErrInvalidChannelFlow, "ICS-27 app version negotiation is unsupported on controller chains")
```

Comment:
Controller submodule returns an error for invoking `NegotiateAppVersion`, for ics27 this should only be provided by the `host` submodule